.github/ISSUE_TEMPLATE/validation-rule-request.md

```diff
@@ -0,0 +1,47 @@
+---
+name        : Validation rule request
+about       : A detailed specification for a new SARIF validation rule to add to the Sarif.Multitool validate command.
+title       : "[RULE REQUEST] Concise description of new analysis"
+labels      : validation-rule-request
+assignees   : ''
+
+---
+
+***********************************************************************************************************
+
+# Rule Proposal: [Friendly Rule Name]
+- *Synopsis*                : [Brief summary of the rule, include code snippets if possible.]
+
+- *[Violation Example]*     : [Optional, Include code snippet which should trigger the violation.]
+
+- *[No Violation Example]*  : [Optional, Include code snippet which demostrates ideal condition (no violation).]
+
+***********************************************************************************************************
+
+### Rule metadata
+- [*Id*]                : [Should be formatted as `SARIF1nnn`, leave blank if unsure]
+- *Name*                : [Provide a friendly symbolic name for the rule in PascalCase.]
+- *Level*               : [Possible values are: `error`, `warning`, or `note`.]
+- *Short description*   : [Short description of the rule.]
```

Comment:
No short description. Taken from the first sentence of the full description.

Reply:
ok. i will include this detail in the instruction so that the rule requester knows.

---
In reply to: [435554545](https://github.com/microsoft/sarif-sdk/pull/1903#discussion_r435554545) [](ancestors = 435554545)

Reply:
good idea.

---
In reply to: [435563131](https://github.com/microsoft/sarif-sdk/pull/1903#discussion_r435563131) [](ancestors = 435563131,435554545)